src/styles/index.css

```diff
@@ -0,0 +1,17 @@
+/* Error: Value null can't be used in a calculation.
+ *    ,
+ * 54 |       margin-left: calc((index($day, ("mon", "tue", "wed", "thu", "fri", "sat", "sun")) - 1) * ($day-width + $gap + 1px));
```

Comment:
The error message indicates that there is a problem with the calculation in the SCSS file. The `index` function might be returning `null`, which cannot be used in a calculation. Ensure that the variable `$day` is defined and has a valid value that exists in the list `("mon", "tue", "wed", "thu", "fri", "sat", "sun")`.